apis/management.cattle.io/v3/kontainer_types.go

```diff
@@ -0,0 +1,42 @@
+package v3
+
+import (
+	"github.com/rancher/norman/condition"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type KontainerDriver struct {
+	metav1.TypeMeta `json:",inline"`
+	// Standard object’s metadata. More info:
+	// https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md#metadata
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+	// Specification of the desired behavior of the the cluster. More info:
+	// https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md#spec-and-status
+	Spec KontainerDriverSpec `json:"spec"`
+	// Most recent observed status of the cluster. More info:
+	// https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md#spec-and-status
+	Status KontainerDriverStatus `json:"status"`
+}
+
+type KontainerDriverStatus struct {
+	ActualURL      string      `json:"actualUrl"`
+	ExecutablePath string      `json:"executablePath"`
+	Conditions     []Condition `json:"conditions"`
+	DisplayName    string      `json:"displayName"`
```

Comment:
should displayName live in spec? i think that's how mapper works to find that field...

Reply:
It should not live in spec because it is a computed field, not settable by the user.  Also, the mapping is working correctly even with it in status instead of spec.